recipes/bioconductor-simpleaffy/build.sh

```diff
@@ -0,0 +1,5 @@
+
```

Comment:
@sebastian-luna-valero can you fix the build.sh files and remove the empty line?